js/apps/system/_admin/aardvark/APP/frontend/js/views/documentsView.js

```diff
@@ -671,6 +671,20 @@
               if (!this.collection.getIsEnterpriseEdge()) {
                 // We do show the _key globally for every type of edge collection,
                 // except in case we do have an "EnterpriseSmart" edge.
+                var documentNameValidations = [
+                  {
+                    rule: Joi.string().regex(/^[a-zA-Z0-9_\-:\.@()\+,=;$!*\%'%]+$/),
```

Comment:
At the end of the sequence I see `\%'%]`. Why is the percent character listed twice, and escaped once? This does not look right.